src/Goerli-DssSpellCollateral.sol

```diff
@@ -96,16 +106,42 @@ contract DssSpellCollateralAction {
 
     function updateCollaterals() internal {
         // ------------------------------- Collateral updates -------------------------------
-
-        // Enable autoline for XXX-A
-        // Poll Link:
-        // Forum Link:
-        // DssExecLib.setIlkAutoLineParameters(
-        //    XXX-A,
-        //    AMOUNT,
-        //    GAP,
-        //    TTL
-        // );
+        uint256 lineReduction;
```

Comment:
```suggestion
        uint256 internal lineReduction;
```

Reply:
This is function scoped, not contract scoped.